User Interface Update

lkiesow opened the pull request:
This patch updates the user interface to make use of the new information
we have due to the database.

The new interface now shows basic statistics as well as a list of
processed and upcoming recordings.

It also updates the design to look more like something related to
Opencast. It vaguely resembles the Opencast admin interface as well as
(or maybe more so) the Opencast REST documentation.

JanKoppe commented on pyca/ui/__init__.py:
```diff
@@ -45,9 +36,27 @@ def home():
+                        .limit(5)
+    recorded_events = db.query(RecordedEvent)\
+                        .order_by(RecordedEvent.start.desc())\
+                        .limit(15)
```
I'm not sure if we should limit this. It could give a false sense about which recordings are actually still on the device.

lkiesow replied:
The table will have a line with an ellipsis at the beginning and/or end if the limit is exceeded. Long-term, I would make the list expandable or introduce paging.

JanKoppe commented on pyca/ui/static/style.css:
```diff
@@ -0,0 +1,87 @@
+    text-align: left;
+}
+
+summary > div > div > div {
```
I think this could be improved. CSS selectors with high specificity are hard to maintain, though it is not really important right now - still managable. Maybe switch over to classes in the future?

lkiesow commented on pyca/__main__.py:
```diff
@@ -110,8 +110,8 @@ def main():
-        import pyca.ui
-        pyca.ui.app.run(host='0.0.0.0')
+        ui.app.run()
+        # pyca.ui.app.run(host='0.0.0.0')
```
Yes, we can. Will do that now.

lkiesow commented:
Updated the pull request and hopefully fixed all issues. I also implemented the limit extensions if there are more recordings than the original limits.
